.github/workflows/publish-and-sync-repos.yml

```diff
@@ -1,11 +1,12 @@
-name: Publish
+name: Publish and Sync Repos
 # Based on: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages
 on:
   push:
-    tags:
-      - 'v*'
+    branches: [main]
```

Comment:
This will trigger all merges to main, below I added a conditional for publish based on the v tag